src/lib/viewers/controls/color-picker/ColorPickerToggle.scss

```diff
@@ -0,0 +1,59 @@
+@import '~box-ui-elements/es/styles/variables';
+@import '../styles';
+
+.bp-ColorPickerToggle {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 48px;
+    height: 48px;
+
+    .bp-ColorPickerToggle-button {
+        position: absolute;
+        width: 48px;
+        height: 48px;
+        padding: 4px 5px;
+        background: transparent;
+        border: none;
+        border-radius: 4px;
+        cursor: pointer;
+
+        svg {
+            background-color: #0061d5;
+            border-radius: 2px;
+            box-shadow: 0 0 0 2pt white;
+        }
+    }
+
+    // &.bp-is-active {
+    //     svg {
+    //         background-color: $white;
+    //         fill: $black;
+    //     }
+    // }
+
+    .bp-ControlsBar {
```

Comment:
We should try to avoid re-styling the control bars. Can we rely on the default styling and absolutely position the palette above the control bar?

Reply:
Yep I'll give that a try.